src/throttledStore.tsx

```diff
@@ -279,17 +280,19 @@ export const createThrottledStore = (
         observableNotify: onObservableNotify,
         resetPendingNotifications: resetAllPendingNotifications,
         hasPendingSubscribers: () => pendingBroadcastNotification,
-        deferSubscriberNotifications: async action => {
+        deferSubscriberNotifications: async (action, shouldClearMemoizedForState) => {
```

Comment:
Shouldn't this flag be used in the `flushMemoizedForState` as well?
[Here](https://github.com/wix-incubator/repluggable/pull/261/files#diff-e2b9f6a4b29afca4f5f8e1a1dab7911eb2c2a775a38af23e3272ce00683c72edR682)

Reply:
I think my names were wrong, see what you think now

Reply:
Oh
Got it :)